frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -350,7 +350,7 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
 
   // Step 2 -------------------------------------------------------
 
-  hasRevelation: boolean;
+  hasSuper: boolean;
```

Comment:
I think it's fine for the type to be `SUPER`, but the var names and function names it could be more clear as `SuperBallet`, for example `hasSuperBallet`.